cfnet/_ckpt_manager.py

```diff
@@ -51,6 +51,8 @@ def update_checkpoints(self,
                            steps: Optional[int] = None):
         if self.monitor_metrics is None:
             return
+        if self.monitor_metrics not in epoch_logs:
+            raise ValueError('The monitor_metrics is not appropriately configured.')
```

Comment:
Change to 
'''python
"The monitor_metrics (`{}`) is not appropriately configured.".format(self.monitor_metrics)
'''